frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1573,6 +1587,8 @@ impl<T: Config> Pallet<T> {
 		// Phase is off now.
 		<CurrentPhase<T>>::put(Phase::Off);
 
+		Self::deposit_event(Event::OffPhaseStarted { round: Self::round() });
```

Comment:
Looking again, I feel this is not needed. `OffPhaseStarted` is similar to saying "start of the new round" (`NewRoundStarted` might be a better name). This always happens in the same block as `ElectionFinalised` event and is redundant.